vectorsearch/README.md

```diff
@@ -0,0 +1,178 @@
+# Vector Search Workload
+
+This workload is to benchmark performance of indexing and search of Vector Engine of Opensearch.
+
+## Datasets
+
+This workload currently supports datasets  with either HDF5 format or Big-ann.
+You can download datasets from [here](http://corpus-texmex.irisa.fr/) to benchmark the quality of approximate k-NN algorithm from
+OpenSearch.
+
+### Running a benchmark
+
+Before running a benchmark, make sure you have the endpoint of your cluster and
+  the machine you are running the benchmarks from, can access it. 
+ Additionally, ensure that all data has been pulled to the client.
+
+Currently, we support one test procedures for the vector search workload: 
+no-train-test that does not have steps to train a model included in the 
+schedule. This test procedures will index a data set 
+of vectors into an OpenSearch index and then run a set of queries against them. 
+
+To run test procedure, open up 
+[params/no-train-params.json](params/no-train-params.json) and 
```

Comment:
We're missing the params file `no-trains-params.json`. 
<img width="312" alt="image" src="https://github.com/opensearch-project/opensearch-benchmark-workloads/assets/51065478/a39e3251-db66-4069-a8bd-fd25e66c9740">

It might be more clear to say something like this:  
```
To run the test procedure, create a file based off of the contents of either `faiss-sift-128-l2.json`or `nmslib-sift-128-l2.json` and modify the parameters. Once parameters are set in the file, export the URL, PORT, TEST PROCEDURE, and params file you just created. Run the following command to run the no trains test procedure
```